sdk/formrecognizer/ai-form-recognizer/samples-dev/recognizeBusinessCard.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to recognize elements of a business card from
+ * a file using a prebuilt model.
+ *
+ * The prebuilt business card model can return several fields. For a detailed
+ * list of the fields supported by the business card model, see the following
+ * link:
+ *
+ * https://aka.ms/azsdk/formrecognizer/businesscardfields
```

Comment:
NIT: in some places you are using `https://aka.ms/formrecognizer/businesscardfields` and in others `https://aka.ms/azsdk/formrecognizer/businesscardfields`. Both work, but wanted to call it out in case you want consistency